data/Cowling_1987/metadata.yml

```diff
@@ -0,0 +1,208 @@
+    sands on a gently undulating coastal plain. The climate is dry, warm, mediterranean
+    (Beard 1976). The mean annual rainfall at Dongara, 25 km north of the study area,
+    is 487 mm. At Eneabba, 50 km south of the study area, mean maximum temperatures
+    in January (hottest month) and mean minimum temperatures in July (coldest month)
+    are 35.6 deg C and 9.1 deg C, respectively. Absolute maximum temperatures in excess
+    of 40 deg C are recorded from December to March (Bureau of Meteorology 1959).
+    The vegetation at the study area is a Banksia-dominated scrub-heath with a tall
+    (1.5-4.0 m) shrub stratum over a closed layer of heath shrubs 1-m tall (Beard
+    1976). We chose a homogeneous 4.8-ha area for intensive study. The only other
+    prominent overstorey shrub was Actinostrobus arenarius. Understorey shrubs included
+    species of Beaufortia, Calytrix, Conospermum, Hakea, Leptospermum and Petrophile.
+    At the time of sampling, the vegetation had last been burnt fifteen years ago
+    (Western Australian Bushfires Board, personal communication).
+  original_file: none, transcribed from manuscript
+  notes: none
+locations:
+  Mount Adams:
+    latitude (deg): -29.417
+    longitude (deg): 114.967
+    description: Banksia-dominated scrub-heath
+    soil type: deep, infertile aeolian sand
+    topography: gently undulating coastal plain
+    vegetation type: area is a Banksia-dominated scrub-heath with a tall shrub stratum
+      over a closed layer of heath shrubs
+    fire history (years since fire): 15
+contexts: .na
+traits:
+- var_in: flowering_time
+  unit_in: .na
+  trait_name: flowering_time
+  entity_type: population
+  value_type: mode
+  basis_of_value: measurement
+  replicates: 15
+  methods: The number of open inflorescences was counted on fifteen randomly chosen
+    individuals of each species at monthly intervals between May 1983 and February
+    1984. We also recorded the number of fresh inflorescences beneath each shrub which
+    had been removed from the plant as a result of foraging by white-tailed black
+    cockatoos (Calyptorhyncus funerus latirostris).
+- var_in: resprouting_capacity
+  unit_in: .na
+  trait_name: resprouting_capacity
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: literature, expert knowledge
+- var_in: bud_bank_location
+  unit_in: .na
+  trait_name: bud_bank_location
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: literature, expert knowledge
+- var_in: serotiny
+  unit_in: .na
+  trait_name: serotiny
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: literature, expert knowledge
+- var_in: germination_%
+  unit_in: '%'
+  trait_name: seed_germination
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 45
```

Comment:
I've added cone-specific germination rates, but something seems really odd to me, since as you point out, the overall mean isn't a mean of the individual age classes. I think, based on methods, the overall mean if specified as being a value based on 15 individuals (replicates = 15), while age classes I've now put as 45 (15 x 3).